### PR TITLE
Use tracer utils

### PR DESCRIPTION
**Pull request by dospore**
- Abstract some of the signing logic out so it can be used elsewhere
- Import signing logic from tracer-utils
- Renamed tracer package to tracer-protocol since npm said that tracer was taken :(

### CODE REVIEW

File: migrations-ts/2_deploy_contracts.ts [hunk 1]
```diff
@@ -48,15 +48,16 @@ try {
...
 
     console.log("Writing out contract addresses")
     const obj = {
         'account': account,
         'factory': factory,
         'insurance': insurance,
         'pricing': pricing,
-        'oracle': oracle
+        'oracle': oracle,
+	'trader': trader
```
Comment (sporejack) [inline migrations-ts/2_deploy_contracts.ts:60]: Indentation.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@tracer/contracts",
-  "version": "0.1.0",
+  "name": "@tracer-protocol/contracts",
+  "version": "0.0.1",
```
Comment (sporejack) [inline package.json:3]: Should probably reach consensus on versioning here.

**Review by sporejack — CHANGES_REQUESTED** [review]
LGTM -- minor changes though.

**Comment by raymogg** [comment]
Closed due to inactivity. We can use `tracer-utils` at another point.

**Pull request closed without merging**